src/scores/stats/tests/diebold_mariano_impl.py

```diff
@@ -55,6 +55,9 @@ def diebold_mariano(
     Harvey, Leybourne and Newbold (1997) for why the t distribution may be preferred,
     especially for shorter timeseries.
 
+    If `da_timeseries` is a chunked array, data will be brought into memory during
```

Comment:
This `tests` directory name really confused me! If there's another word for a statistical test, or even `stats_tests`, that might be worth considering renaming it?

Reply:
I agree - I renamed it `statistical_tests`

Reply:
I've now reverted this and created a new issue https://github.com/nci/scores/issues/66 so that the discussion can happen there to prevent this pull request from being delayed.